integrations/pom.xml

```diff
@@ -16,27 +16,21 @@
     <jackson.version>2.14.2</jackson.version>
     <enforcer.skip>true</enforcer.skip> <!-- I give up, too much effort to maintain enforcer here, should probably try clear out all enforcer only deps and see if it works -->
```

Comment:
This PR does satisfy Enforcer (to keep the classpath clean and logical) but I am leaving this line in to ease future maintenance when merging Dependabot PRs.